jan5_assign.html

```diff
@@ -1,32 +1,37 @@
 <script>
   var question, 
-      answer,
-      guess,
-      count,
-      pass;
-    
-  question = ("Can you guess the number of characters that make up this question?");
+    answer,
+    guess,
+    count,
+    pass;
+  
   alert("You have a maximum of 3 attempts to get this right");
+  question = ("Can you guess the number of characters that make up this question?");
+  
   guess = prompt(question);
   answer = question.length;
-
   count = 1;
-
-  while(count < 3){
+  pass = false;
+  
+  while((count < 3) && (pass == false)) {
     if (guess == answer) {
-        alert("Nice work!");
-        pass = true;
+      alert("Nice work!");
+      pass = true;
+    }
+    else if(guess > answer) {
+      guess = prompt("Too high, try again!");
+      count++;
+      pass = false;
     }
     else {
-      guess = prompt("Try again!");
+      guess = prompt("Too low, try again!");
       count++;
```

Comment:
count++ can happen no matter the result of the `if`, so you can have it before or after the if and only once.